app/workers/zomato_data_generator.rb

```diff
@@ -0,0 +1,25 @@
+class ZomatoDataGenerator
+  def self.download_zomato_data(reader)
+    restaurants = reader.fetch_restaurants_list
```

Comment:
I would be explicit as for the data type inside, and call this `restaurants_jsons`